tests/foreman/ui/test_contentview.py

```diff
@@ -5007,8 +5007,7 @@ def test_positive_delete_cv_promoted_to_multi_env(self):
             self.content_views.remove_version_from_environments(
                 cv_name, version, list(all_env_names_set))
             # delete content view
-            self.content_views.delete(cv_name)
-            self.assertIsNone(self.content_views.search(cv_name))
```

Comment:
Yes, we don't need assertion after delete operations as it is performed inside of delete method